packages/client/src/lib/interface.ts

```diff
@@ -75,6 +75,18 @@ export type PutOptions = {
    * Maximum times to retry a failed upload.
    */
   maxRetries?: number
+  /**
+   * Should input files be wrapped with a directory. 
+   * Enabling it preserves the input filenames in DAG as they become
```

Comment:
It is enabled by default. Perhaps Disabling it?

I think we should mention the defaults used (same for maxRetries for example)